app/src/main/java/org/mozilla/fenix/browser/BaseBrowserFragment.kt

```diff
@@ -1185,9 +1185,9 @@ abstract class BaseBrowserFragment : Fragment(), UserInteractionHandler, Activit
     private fun showQuickSettingsDialog() {
         val tab = getCurrentTab() ?: return
         viewLifecycleOwner.lifecycleScope.launch(Main) {
-            val sitePermissions: SitePermissions? = tab.content.url.toUri().host?.let { host ->
-                val storage = requireComponents.core.permissionStorage
-                storage.findSitePermissionsBy(host)
+            val sitePermissions: SitePermissions? = tab.content.url.getOrigin()?.let { origin ->
+            val storage = requireComponents.core.permissionStorage
+                storage.findSitePermissionsBy(origin)
```

Comment:
Shouldn't this be indented inside the let?

Reply:
Yes, you are right, it's weird that `detekt` didn't catch it

Reply:
Addressed!